README.md

```diff
@@ -102,3 +102,31 @@ order to create everything with the right parameters. E.g.:
 There are a number of env vars this script looks at which are useful
 when running directly; check the script headers for details.
 
+## Throttling logs in Fluentd
+
+For projects that are especially verbose, an administrator can now throttle
+down the rate at which the logs are read in.  To tell Fluentd which projects
+it should be restricting you will need to do the following:
+
+Create a yaml file that contains project names and the desired rate at which
+logs are read in. (Default is 1000)
+```
+logging:
+  read_lines_limit: 500
+
+test-project:
+  read_lines_limit: 10
+
+.operations:
+  read_lines_limit: 100
+```
+
+Create a secret providing this file as the source
+```
+oc secrets new fluentd-throttle settings=</path/to/your/yaml>
+```
+
+Mount the created secret to your Fluentd container
+```
+oc volumes dc/logging-fluentd --add --type=secret --secret-name=fluentd-throttle --mount-path=/etc/throttle-settings --name=throttle-settings --overwrite
```

Comment:
I'm a little surprised... doesn't the secret need to be added to the fluentd service account too?